typescript-babel-combination/test/index.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { equal } from 'assert'
+import { hello } from '../src'
+
+describe('Typescript + Babel usage suite', () => {
+  it('should be able to execute a test', () => {
+    equal(true, true)
```

Comment:
We don't really need this. Sorry as I know its in the TS example, we should remove that one too.
I'd rather we only had 1 test than have something just for the sake of it.

Reply:
Totally agree. I will remove this one.